BookingsApi/BookingsApi.Contract/Responses/BookingsHearingResponse.cs

```diff
@@ -121,5 +121,10 @@ public DateTime HearingDate
         /// Gets or sets Judge/Court account (username).
         /// </summary>
         public string CourtRoomAccount { get; set; }
+        
+        /// <summary>
+        /// Gets or sets Allocated VHO.
+        /// </summary>
+        public string AllocatedVho { get; set; }
```

Comment:
It's actually CSO, not VHO. Perhaps we can instead call this `AllocatedTo` for simplicity (and it also matches what we call it in the UI too).

Reply:
done